src/com/comeet/UserService.java

```diff
@@ -93,6 +93,7 @@
      */
     @GET
     @Path("/{orgDomain}/users/{user}/meetings")
+    @Produces("application/json")
     public List<Meeting> getUserMeetings(@Context HttpHeaders headers,
                     @PathParam("orgDomain") String orgDomain, @PathParam("user") String user,
                     @DefaultValue("") @QueryParam("start") String start,
```

Comment:
Don't we need an end date for the meeting? I am pretty sure we don't want to return their whole calendar